apps/web/src/components/dashboard/energy-cost.tsx

```diff
@@ -92,7 +91,8 @@ export function getPredictedCost(userData: UserDataSelectType[], energyData: Sen
         return 0;
     }
 
-    const price = getLatestUserData(userData).basePrice;
+    // Hier wird sichergestellt, dass 'price' einen Wert hat oder auf 0 gesetzt wird, falls es 'null' oder 'undefined' ist.
```

Comment:
Kommentare sollten auf englisch sein. Hier finde ich den Kommentar aber eh überflüssig, da der Code in der nächsten Zeile ja selbst verständlich ist. Ich würde den Kommentar also eher weglassen

Reply:
Sorry hatte vergessen den Kommentar rauszunehmen. Der war nur für mich gedacht